app/components/Modal/DepositModal.jsx

```diff
@@ -182,6 +209,27 @@ class DepositModalContent extends DecimalChecker {
                 },
                 fetchingAddress: false
             });
+        } else if (
```

Comment:
This section should not be hard-coded for a specific gateway. We should specify a type of gateway (ie, SimpleAssetGateway was one) and make sure it's followed by everyone using that way.

Please change ChainStore.getAccount to a non-subscribing, or use `FetchChain()`

Reply:
changed